sympy/core/add.py

```diff
@@ -610,6 +610,21 @@ def as_numer_denom(self) -> tuple[Expr, Expr]:
 
         return _keep_coeff(ncon, n), _keep_coeff(dcon, d)
 
+    @property
+    def is_hermitian(self):
+        from sympy.simplify import simplify
+        if simplify(self.func(*self.args) - self._eval_adjoint()).is_zero:
+            return True
+        return
+
+    @property
+    def is_antihermitian(self):
+        from sympy.simplify import simplify
+        if simplify(self.func(*self.args) + self._eval_adjoint()).is_zero:
+            return True
+        return
```

Comment:
Let's not have this change mixed in with the bigger change of removing hermitian from the assumptions system.